docs/resources/grant_privileges_to_account_role.md

```diff
@@ -0,0 +1,362 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "snowflake_grant_privileges_to_account_role Resource - terraform-provider-snowflake"
+subcategory: ""
+description: |-
+  
+---
+
+
+!> **Warning** Be careful when using `always_apply` field. It will always produce a plan (even when no changes were made) and can be harmful in some setups. For more details why we decided to introduce it to go our document explaining those design decisions (coming soon).
```

Comment:
I think it would be nice to add here the same communication I have suggested for the previous resource: https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/2306#discussion_r1441653749 (and I also missed that it was not added there, so please cover both resources)